infra/configuration.nix

```
@@ -0,0 +1,86 @@
+{ pkgs, lib, ... }:
+let
+  # Note: this might be surprising not to reuse the parent npins/ directory.
+  # The rationale is that the staging environment's sources are decorrelated from the development's sources.
+  # Sources here are managed on a different lifecycle and have different acceptance tests than the development.
+  # Also, the focus in the staging environment is a secure deployment, which trumps over dirty hacks.
+  sources = import ./npins;
+in
+{
+  imports = [
+    "${sources.agenix}/modules/age.nix"
+    ./sectracker.nix
+  ];
+  networking.hostName = "stfmaster";
```

```suggestion
  networking.hostName = "sectracker";
```

I recommend something along these lines. While the Sovereign Tech Fund (STF) made all of this possible to begin with by funding the project, the service is the Nixpkgs security tracker.

---

Agreed and nice catch, applied.